app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+# class User < ApplicationRecord
+#   has_many :posts
+#   has_many :likes
+#   has_many :comments
+# end
```

Comment:
Please be sure to remove commented lines from your code so that your project is as clean and professional as it can 👀